media-libs/svt-av1/svt-av1-0.8.7.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic
+
+DESCRIPTION="Scalable Video Technology for AV1 (SVT-AV1 Encoder and Decoder)"
+HOMEPAGE="https://gitlab.com/AOMediaCodec/SVT-AV1"
+
+if [[ ${PV} = 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/AOMediaCodec/SVT-AV1.git"
+else
+	SRC_URI="https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v${PV}/SVT-AV1-v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc -x86" # -x86: https://github.com/AOMediaCodec/SVT-AV1/issues/1231
+	S="${WORKDIR}/SVT-AV1-v${PV}"
+fi
+
+# Also see "Alliance for Open Media Patent License 1.0"
+LICENSE="BSD-2 Apache-2.0 BSD ISC LGPL-2.1+ MIT"
+SLOT="0"
+
+DEPEND="amd64? ( dev-lang/yasm )"
```

Comment:
Shouldn't this be BDEPEND? I don't believe it's using libyasm.

Beside this nitpick, I see no issues here -- 9999 still builds too.

Reply:
fixed (finally)